src-ui/src/app/components/manage/settings/settings.component.ts

```diff
@@ -194,6 +195,9 @@ export class SettingsComponent
       notificationsConsumerSuppressOnDashboard: this.settings.get(
         SETTINGS_KEYS.NOTIFICATIONS_CONSUMER_SUPPRESS_ON_DASHBOARD
       ),
+      savedViewsWarnOnClose: this.settings.get(
+        SETTINGS_KEYS.SAVED_VIEWS_WARN_ON_UNSAVED_CHANGE
```

Comment:
Any reason to have these named differently? savedViewsWarnOnClose vs SAVED_VIEWS_WARN_ON_UNSAVED_CHANGE

Reply:
fixed =)